README.md

```diff
@@ -69,10 +72,11 @@ Alternatively, you can organize your login information with file structure. For
   * mysite.com
 
 PassFF will
+* get the user from the "user" file unser supersite.com
```

Comment:
This is wrong, but the documentation was misleading here. Would you please change the above example to include a `* password` and a `* url` line and remove the `* user` line?
```
* www
  * supersite.com
    * login
    * password
    * url
  * mysite.com
```
```suggestion

```